go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/opencontainers/image-spec v1.0.1 // indirect
 	github.com/opencontainers/runc v0.1.1 // indirect
 	github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2
+	github.com/pkg/errors v0.8.1
```

Comment:
I don't think this import was intentional. We can use the golang errors package and don't need that one.

Reply:
fixed.